ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -174,6 +174,9 @@ def main(argv=sys.argv[1:]):
             if os.path.samefile(key, filename):
                 filename = key
                 break
+        # report errors from additional include directories
+        if filename not in report:
+            report[filename] = []
```

Comment:
How about making `report` a `defaultdict` instead?

Reply:
It used to be, but caused a regression: https://github.com/ament/ament_lint/pull/134

We could still change it back to a defaultdict, as long as we keep the explicit initialization at L162-163.

Reply:
Using defaultdict de12f2e